src/test/kotlin/com/example/serverFactoryAPI/controllers/AdminControllerIntTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.serverFactoryAPI.controllers
+
+import org.junit.jupiter.api.Test
+import com.example.serverFactoryAPI.util.LogInUtils
+import org.junit.Before
+import org.junit.runner.RunWith
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.http.MediaType
+import org.springframework.security.core.context.SecurityContextHolder
+import org.springframework.test.context.junit4.SpringRunner
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers
+
+@RunWith(SpringRunner::class)
+@SpringBootTest
+@AutoConfigureMockMvc
+class AdminControllerIntTest {
+
+    @Autowired
+    lateinit var mockMvc: MockMvc
```

Comment:
@MarkoLazovicCode As I mentioned in one previous question, this is most likely the answer for having: lateinit var(s).